### PR TITLE
[rust/de] [rust/en] [rust/es] [rust/fr] [rust/cn] Fix comment in Rust docs: box -> mine

### PR DESCRIPTION
**Pull request by Rob--W**
- [x] PR touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] YAML Frontmatter formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Seriously, look at it now. Watch for quotes and double-check field names.

### CODE REVIEW

File: fr-fr/rust-fr.html.markdown [hunk 1]
```diff
@@ -285,7 +285,7 @@ fn main() {
     var = 3;
     let ref_var: &i32 = &var;
 
-    println!("{}", var); // Contrairement `box`, `var` peut encore être utilisé
+    println!("{}", var); // Contrairement `mien`, `var` peut encore être utilisé
```
Comment (vendethiel) [inline fr-fr/rust-fr.html.markdown]: typo

Comment (Rob--W) [inline fr-fr/rust-fr.html.markdown]: That is not a typo, that is the name of the variable at line 267.

Comment (vendethiel) [inline fr-fr/rust-fr.html.markdown]: ugh. i really did a bad job of proofreading that one :-(.

**Comment by vendethiel** [comment]
Thanks.

**Pull request merged**